### ISSUE TITLE
add renovatebot for check latest compatibility

### ISSUE THREAD

**Issue by hardfist** [web-infra-dev/rspack-compat#8]

**Comment by bvanjoi** [comment]
This is unnecessary as daily checks will be conducted within the `ecosystem-ci`.

**Comment by hardfist** [comment]
> This is unnecessary as daily checks will be conducted within the `ecosystem-ci`.
we cant check latest version in eco ci because of lock

**Comment by bvanjoi** [comment]
> we cant check latest version in eco ci because of lock

No, lateset version is injected as an override: https://github.com/web-infra-dev/rspack-ecosystem-ci/blob/main/utils.ts#L455
